Move docs to Readthedocs

Simonmicro opened the pull request:
This implements #82 

I corrected a ton of grammar errors and formatting problems with Sphinx. I also was so free to merge some sections of the readme into the docs and expanded them accordingly.

Demo is online at https://py-kms-demo.readthedocs.io/en/readthedocs/readme.html

@SystemRage To provide an official doc page _someone_ has to register an own account and create the official `py-kms` documentation page. As always, I could do this for you and add you as maintainer later on - or you could do this and may add myself, so I could further maintain the page (completely optional). After this has been setup I'll create one further PR, which changes the links from the demo to the official one. If you need any help - don't hesitate to ask!

I really hope this huge change is welcome :grin: !

SystemRage commented:
Created https://py-kms.readthedocs.io/en/latest/
and i added you as maintainer.
So you can point your demo to this new doc page.

Simonmicro commented:
@SystemRage All clear - I took a look into the build settings and they seem to be fine. All whats left now is to merge this PR, wait a couple of minutes and hopefully enjoy the new documentation! I already updated the urls inside the repo to point to the new docs - lets hope all goes smooth :smile: 
I also modified the project settings to notify me on any doc failures...
